projects/distributed-tracing/src/shared/dashboard/widgets/table/table-widget.model.ts

```diff
@@ -102,6 +102,10 @@ export class TableWidgetModel extends TableWidgetBaseModel {
     return this.filterOptions;
   }
 
+  public getRowSelectionHandlers(): TableWidgetRowSelectionModel[] {
```

Comment:
Can we not access `this.rowSelectionHandlers` property directly? What do we gain by polymorphism here?

Reply:
I am not up to date with the new design. So feel free to ignore. I was thinking If model property is defined in the base model, then it should be made available without the function